frontend/pages/shopping-lists/_id.vue

```diff
@@ -101,6 +164,45 @@
         </v-card>
       </BaseDialog>
 
+      <!-- Menu dialog -->
```

Comment:
This needs still some work. Having a button in a three dot menue that leads to a modal with further menue options is not the way to do it.